### PR TITLE
Remove spinner from create shipping label button

### PR DESCRIPTION
**Pull request by Ferdev**
Removes the preloading step in the create shipping label component and provides it with all the that it needs from the PHP side.

### CODE REVIEW

File: classes/class-wc-connect-shipping-label.php [hunk 1]
```diff
@@ -39,11 +54,22 @@ class WC_Connect_Shipping_Label {
...
 			WC_Connect_API_Client $api_client,
 			WC_Connect_Service_Settings_Store $settings_store,
-			WC_Connect_Service_Schemas_Store $service_schemas_store
+			WC_Connect_Service_Schemas_Store $service_schemas_store,
+			WC_Connect_Payment_Methods_Store $payment_methods_store
 		) {
 			$this->api_client = $api_client;
 			$this->settings_store = $settings_store;
 			$this->service_schemas_store = $service_schemas_store;
+			$this->payment_methods_store = $payment_methods_store;
```
Comment (superdav42) [inline classes/class-wc-connect-shipping-label.php:63]: This property appears to be unused by the class WC_Connect_Shipping_Label so it should be safe to remove.

File: classes/class-wc-connect-shipping-label.php [hunk 2]
```diff
@@ -39,11 +54,22 @@ class WC_Connect_Shipping_Label {
...
+			$this->account_settings = new WC_Connect_Account_Settings(
+					$settings_store,
+					$payment_methods_store
+			);
+			$this->package_settings = new WC_Connect_Package_Settings(
+					$settings_store,
+					$service_schemas_store
+			);
+			$this->continents = new WC_Connect_Continents();
```
Comment (superdav42) [inline classes/class-wc-connect-shipping-label.php:72]: Next hack week let's setup [PHP-DI](http://php-di.org/) so we don't have to do all this stuff.

File: classes/class-wc-connect-shipping-label.php (hunk 1 above)
Comment (Ferdev) [reply classes/class-wc-connect-shipping-label.php]: oh yes, you're right!

File: classes/class-wc-connect-account-settings.php [hunk 3]
```diff
@@ -0,0 +1,58 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( class_exists( 'WC_Connect_Account_Settings' ) ) {
+	return;
+}
```
[the remark this answers is not in the document]
Comment (Ferdev) [reply classes/class-wc-connect-account-settings.php]: I was wondering why these checks were all over the place. I'll remove them in the classes I've introduced in this PR.

File: classes/class-wc-connect-shipping-label.php (hunk 2 above)
Comment (Ferdev) [reply classes/class-wc-connect-shipping-label.php:66]: Yeah, let's do that. I'm a big fan of dependency injection.

**Pull request merged**